docs/known-issues.md

```diff
@@ -123,4 +123,40 @@ Enabling Mandatory ASLR affects the MSYS2 core library, which is relied upon by
 
 Electron enables hardware accelerated graphics by default, but some graphics cards have issues with hardware acceleration which means the application will launch successfully but it will be a black screen.
 
-**Workaround:** if you set the `GITHUB_DESKTOP_DISABLE_HARDWARE_ACCELERATION` environment variable to any value and launch Desktop again it will disable hardware acceleration on launch, so the application is usable.
\ No newline at end of file
+**Workaround:** if you set the `GITHUB_DESKTOP_DISABLE_HARDWARE_ACCELERATION` environment variable to any value and launch Desktop again it will disable hardware acceleration on launch, so the application is usable.
+
+### Failed to open CA file after an update - #4832
+
+A recent upgrade to Git for Windows changed how it uses `http.sslCAInfo`.
+
+An example of this error:
+
+> fatal: unable to access 'https://github.com/\<owner>/\<repo>.git/': schannel: failed to open CA file 'C:/Users/\<account>/AppData/Local/GitHubDesktop/app-1.2.2/resources/app/git/mingw64/bin/curl-ca-bundle.crt': No such file or directory
+
+This is occuring because some users have an existing Git for Windows installation that created a special config at `C:\ProgramData\Git\config`, and this config may contain a `http.sslCAInfo` entry, which is inherited by Desktop.
+
+There's two problems with this current state:
+
+ - Desktop doesn't need custom certificates for it's Git operations - it uses SChannel by default, which uses the Windows Certificate Store to verify server certificates
+ - this `http.sslCAInfo` config value may resolve to a location or file that doesn't exist in Desktop's Git installation
+
+**Workaround:**
+
+1. Verify that you have the problem configuration by checking the output of this command:
+
+```
+> git config -l --show-origin
+```
+
+You should have an entry that looks like this:
+
+```
+file:"C:\ProgramData/Git/config" http.sslcainfo=[some value here]
+```
+
+2. Open `C:\ProgramData\Git\config` and remove the corresponding lines that look like this:
```

Comment:
Should we add a line in this section about needing admin privileges to edit this file?

Reply:
Probably, yes